client/packages/coldchain/src/common/ImportFridgeTag.tsx

```diff
@@ -3,7 +3,11 @@ import { LoadingButton, useConfirmationModal } from '@common/components';
 import { UploadIcon } from '@common/icons';
 import { useTranslation } from '@common/intl';
 import { AppRoute, Environment } from '@openmsupply-client/config';
-import { useConfirmOnLeaving, useNotification } from '@common/hooks';
+import {
+  useConfirmOnLeaving,
+  useIsExtraSmallScreen,
```

Comment:
As discussed we probably should get rid of `isMobile` and/or possibly just change to using the media query directly in the components pending a discussion